src/components/OrgView/GlobalSettingsButton.js

```diff
@@ -0,0 +1,185 @@
+import React, { useRef, useState, useCallback } from 'react'
+import PropTypes from 'prop-types'
+import {
+  ButtonBase,
+  ButtonIcon,
+  GU,
+  IconSettings,
+  Popover,
+  useTheme,
+} from '@aragon/ui'
+import IconNetwork from './IconNetwork'
+import IconCustomLabels from './IconCustomLabels'
+import IconNotifications from './IconNotifications'
+import IconHelpAndFeedback from './IconHelpAndFeedback'
+
+function GlobalSettingsButton({ onOpen }) {
+  const theme = useTheme()
+  const [opened, setOpened] = useState(false)
+  const containerRef = useRef()
+
+  const handleToggle = useCallback(() => {
+    setOpened(!opened)
+  }, [setOpened, opened])
+  const handleClose = useCallback(() => {
+    setOpened(false)
+  }, [setOpened])
+  const handleItemClick = useCallback(
+    path => () => {
+      setOpened(false)
+      onOpen(path)
+    },
+    [setOpened, onOpen]
+  )
+
+  return (
+    <React.Fragment>
+      <div ref={containerRef}>
+        <ButtonIcon
+          onClick={handleToggle}
+          css={`
+            height: 100%;
+            width: 40px;
+          `}
+          label="Global preferences"
+        >
+          <IconSettings />
+        </ButtonIcon>
+      </div>
+      <Popover
+        placement="bottom-end"
+        onClose={handleClose}
+        visible={opened}
+        opener={containerRef.current}
+      >
+        <ul
+          css={`
+            width: 260px;
+            padding: 0;
+            margin: 0;
+            list-style: none;
+            background: ${theme.surface};
+            color: ${theme.content};
+            border-radius: 4px;
+          `}
+        >
+          <li
+            css={`
+              display: flex;
+              align-items: center;
+              height: 32px;
+              color: ${theme.surfaceContentSecondary};
+              text-transform: uppercase;
+              font-size: 12px;
+              padding-left: ${2 * GU}px;
+              border-bottom: 1px solid ${theme.border};
+            `}
+          >
+            Global preferences
+          </li>
+          <li
+            css={`
+              border-bottom: 1px solid ${theme.border};
+            `}
+          >
+            <ItemMemo
+              onClick={handleItemClick('custom-labels')}
```

Comment:
`ItemMemo` will rerender every time because these functions are recreated at every render (and we also pass an element to `icon`). It’s probably ok to rerender here, so maybe we could at least remove the `React.memo()`, so that it doesn’t do checks that always end up rerendering?